.github/workflows/test-static-tests.yml

```diff
@@ -31,17 +31,18 @@ jobs:
           mkdir drupal
           cd drupal
           cp ${GITHUB_WORKSPACE}/tests/fixtures.drainpipe-external-test-project/composer.json .
+          cp ${GITHUB_WORKSPACE}/tests/fixtures.drainpipe-external-test-project/Taskfile.yml .
           composer install
       - name: Run static tests
         run: |
           cd ../drupal
-          ./vendor/bin/task -t Taskfile.dev.yml test:static
```

Comment:
I feel like this is the moment where we need a CHANGELOG.md given this may require changes in CI scripts that call into Task.

Reply:
Actually, do we also want a message set if `Taskfile.dev.yml` exists in the repo? Otherwise, I don't see how it would be cleaned up.

Reply:
@deviantintegral I think we're the only two people using it at the moment :smile: